### ISSUE TITLE
can't get this to work with neovim lsp

### ISSUE THREAD

**Issue by Richardk2n** [python-lsp/pylsp-mypy#38]
hi @Richardk2n , I can't get this to work with neovim lsp. I installed with nvim-lsp-installer . Here is my lspconfig:

```lua

lspconfig.pylsp.setup {
  on_attach = on_attach,
  capabilities = capabilities,
  settings = {
    pylsp = {
      configurationSources = { 'flake8' },
      plugins = {
        flake8 = { enabled = true },
        black = { enabled = true },
        pylsp_mypy = { enabled = true, live_mode = true },
        pycodestyle = { enabled = false },
        pyflakes = { enabled = false },
        mccabe = { enabled = false },
      },
    },
  },
}

```

PyLSP version is 1.5.0, and pylsp-mypy is 0.6.2.
Flake8 and black work as expected, but there is no diagnostic for mypy

_Originally posted by @sandangel in https://github.com/python-lsp/pylsp-mypy/issues/35#issuecomment-1186905008_
